insights/combiners/smt.py

```diff
@@ -0,0 +1,64 @@
+"""
+Simultaneous Multithreading (SMT) combiner
+==========================================
+
+Combiner for Simultaneous Multithreading (SMT). It uses the results of the following parsers:
+:class:`insights.parsers.smt.CpuCoreOnline`,
+:class:`insights.parsers.smt.CpuSiblings`.
+"""
+
+from insights.core.plugins import combiner
+from insights.parsers.smt import CpuCoreOnline, CpuSiblings
+
+
+@combiner(CpuCoreOnline, CpuSiblings)
+class CpuTopology(object):
+    """
+    Class for collecting the online/siblings status for all CPU cores.
+
+    Sample output of the ``CpuCoreOnline`` parser is::
+        [[Core 0: Online], [Core 1: Online], [Core 2: Online], [Core 3: Online]]
+
+    Sample output of the ``CpuSiblings`` parser is::
+        [[Core 0 Siblings: [0, 2]], [Core 1 Siblings: [1, 3]], [Core 2 Siblings: [0, 2]], [Core 3 Siblings: [1, 3]]]
```

Comment:
There should be a blank line after lines 19 and 22 to properly format the sample output.  See your generated docs